src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -1271,6 +1271,16 @@ def init(self):
         self.stored_problems = {}
         self.state = SearchingForMNPlusOne(self.settings, self.event())
 
+    def is_creation_allowed_for_organization(self, organization: Optional[Organization]) -> bool:
+        return features.has(
+            "organizations:performance-issues-m-n-plus-one-db-detector",
+            organization,
+            actor=None,
+        )
+
+    def is_creation_allowed_for_project(self, project: Project) -> bool:
+        return True  # Detection always allowed by project for now
```

Comment:
If you want to control detection at a project level, you can also add a project option for that too,
ref https://github.com/getsentry/sentry/pull/43701